lib/kb/models/pet_contract.rb

```diff
@@ -23,10 +23,13 @@ def self.attributes_from_response(response)
                        conversion_utm_source conversion_utm_term
                        conversion_utm_adgroup_id conversion_utm_campaign_id].freeze
     DATE_FIELDS = %i[policy_start_date policy_expiration_date].freeze
-    INTEGER_FIELDS = %i[price_yearly price_monthly price_discount_yearly].freeze
+    INTEGER_FIELDS = %i[
+      price_yearly price_monthly price_discount_yearly payment_interval_months
+    ].freeze
     FIELDS = [*STRING_FIELDS, *DATE_FIELDS, *INTEGER_FIELDS].freeze
 
-    IMMUTABLE_FIELDS = (FIELDS - %i[status contract_document policy_expiration_date]).freeze
+    # TODO: constant not used. Should be removed?
```

Comment:
I'm not sure what to do with this constant. @malkovro I saw that it was used at some point, but it's no longer referenced.

Reply:
Looks like I dropped it myself in https://github.com/barkibu/kb-ruby/commit/e8ca2f8796a9e296bd713259fefbda71b53a1a31... Don't really know why but looks like it was useless if it did not annoy us since then so feel free to drop the constnat !

Reply:
Turns out this constant is referenced by web when syncing the contracts 😄